app/src/main/java/org/dhis2/uicomponents/map/camera/CameraExtension.kt

```diff
@@ -12,17 +12,18 @@ import com.mapbox.mapboxsdk.camera.CameraUpdateFactory
 import com.mapbox.mapboxsdk.geometry.LatLng
 import com.mapbox.mapboxsdk.geometry.LatLngBounds
 import com.mapbox.mapboxsdk.maps.MapboxMap
+import org.dhis2.uicomponents.map.geometry.bound.GetBoundingBox
 
-fun MapboxMap.initCameraToViewAllElements(context: Context, bounds: LatLngBounds) {
+fun MapboxMap.initCameraToViewAllElements(context: Context?, bounds: LatLngBounds) {
     if (bounds.latNorth == 0.0 && bounds.latSouth == 0.0 &&
         bounds.lonEast == 0.0 && bounds.lonWest == 0.0
     ) {
         this.cameraPosition = CameraPosition.Builder()
             .zoom(2.0)
             .build()
-        Toast.makeText(context, "No data to load on map", LENGTH_LONG).show()
+        context?.let { Toast.makeText(context, "No data to load on map", LENGTH_LONG).show() }
     } else {
-        this.easeCamera(CameraUpdateFactory.newLatLngBounds(bounds, 50), 1200)
+        this.easeCamera(CameraUpdateFactory.newLatLngBounds(bounds, 500), 1200)
```

Comment:
What does 500 and 1200 mean in here?

Reply:
padding around the bounds and duration of the animation

Reply:
Maybe we can add a descriptive names to this numbers so that everyone know what they are???